test/k8sT/manifests/bookinfo-v1-istio.yaml

```diff
@@ -172,7 +172,7 @@ spec:
         - "9080"
         - -d
         - "15020"
-        image: docker.io/istio/proxy_init:1.1.2
+        image: docker.io/istio/proxy_init:1.1.3
```

Comment:
As part of this PR, please do the following. 

* Open a PR against `packer-ci-build` and add the new images added in any `test/k8s/manifests/*.yaml` file that was changed as part of this PR. The file to change is `provision/pull-images.sh`. 
* Once that is done, build a new version of the box via logging into Jenkins, going to the [Vagrant-Master-Boxes-Packer-Build job](https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build/), and clicking `Build With Parameters` to build a new version of the box.
* Update the version of the VM used in `Vagrantfile` and `test/Vagrantfile`.

This ensures that the VMs have the images cached for use in the CI.

Reply:
Packer CI PR created (pr/jrajahalme/istio-update-1.1.3). Apparently I do not have build rights on this repo, though.

Reply:
I merged your PR, and built the image :D 

https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build/147/

You can update the Vagrantfiles to use version 147.

Reply:
Done!